From: Kylene Jo Hall <kjhall@us.ibm.com>
Subject: tpm module needs a license (145954)
Patch-mainline: 2.6.16-rc2

Attempting to insert the tpm modules fails because the tpm_bios file is
missing a license statement.

Signed-off-by: Kylene Hall <kjhall@us.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/char/tpm/tpm_bios.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN drivers/char/tpm/tpm_bios.c~tpm-tpm-bios-fix-module-license-issue drivers/char/tpm/tpm_bios.c
--- devel/drivers/char/tpm/tpm_bios.c~tpm-tpm-bios-fix-module-license-issue	2006-01-23 14:57:45.000000000 -0800
+++ devel-akpm/drivers/char/tpm/tpm_bios.c	2006-01-23 14:57:45.000000000 -0800
@@ -547,3 +547,4 @@ void tpm_bios_log_teardown(struct dentry
 		securityfs_remove(lst[i]);
 }
 EXPORT_SYMBOL_GPL(tpm_bios_log_teardown);
+MODULE_LICENSE("GPL");
_
